source/declarators.tex

```diff
@@ -2388,11 +2388,11 @@
 \indextext{initialization|(}
 
 \pnum
-A declarator can specify an initial value for the
+An \grammarterm{init-declarator} can specify an initial value for the
```

Comment:
If we're going to switch from the English term to the grammar term here, I think we should use "*init-declarator* or *member-declarator*". We would also need to do something about *condition*s, which don't explicitly say that the variable is initialized from its *brace-or-equal-initializer*.

Maybe it would be better to say something like:

> A *declarator* can be followed by an *initializer* that specifies an initial value for the entity being declared. [Note: In some contexts, only the *brace-or-equal-initializer* form of *initializer* is permitted by the grammar, but these rules still apply.]